generated/nidcpower/nidcpower/lcr_load_compensation_spot.py

```diff
@@ -0,0 +1,145 @@
+import ctypes
+import numbers
+import platform
+
+import nidcpower._visatype
+import nidcpower.enums as enums
+
+
+# This class is an internal implementation detail
+# ctypes definition
+# Name must match exactly what the name of the structure type is named in the C API.
```

Comment:
This comment is inaccurate. This is the struct:

```
   typedef struct NILCRLoadCompensationSpot_struct
   {
      ViReal64 frequency;
      ViInt32 referenceValueType;
      ViReal64 referenceValueA;
      ViReal64 referenceValueB;
   } NILCRLoadCompensationSpot;
```

Reply:
I also don't understand why they **must match exactly**. Can you explain this?

Reply:
This comment was actually copied from https://github.com/ni/nimi-python/blob/master/src/niscope/custom_types/waveform_info.py#L8.

I have removed the comment "Name must match exactly what the name of the structure type is named in the C API" in my latest commit as I don't think the names must match exactly either.